apps/mobile/src/components/amount-field/amount-field.utils.ts

```diff
@@ -0,0 +1,77 @@
+import { decimalSeparator } from '@/features/send-refactor/temporary-constants';
+import { type CurrencyMode } from '@/features/send-refactor/types';
+
+import { currencyDecimalsMap } from '@leather.io/constants';
+import { type Currency, type MarketData, type Money } from '@leather.io/models';
+import { baseCurrencyAmountInQuote, createMoneyFromDecimal } from '@leather.io/utils';
+
```

Comment:
We need to add unit tests for these amount functions

Reply:
Agreed. Kind of related to your Send utils comment. At least the `baseCurrencyAmountInQuote` doesn't belong here. I'll be re-working some of this when adding fiat mode, will add tests then to avoid doing it twice. Added a note in the issue: [LEA-2382: Enable Fiat entry](https://linear.app/leather-io/issue/LEA-2382/enable-fiat-entry)